ts/components/wallet/WalletLayout.tsx

```diff
@@ -119,6 +167,13 @@ export default class WalletLayout extends React.Component<Props> {
           </Content>
           {this.props.children}
         </ScrollView>
+
+        <Animated.View
+          style={[styles.animatedSubHeader, { height: subHeaderHeight }]}
```

Comment:
Have you tried to create the animation using the translateY.
The problem animating the height is that you can't use the nativeDriver.